Mce/util.py

```diff
@@ -749,8 +749,15 @@ def parse_new_issuance_metadata_10007(data):
             continue
 
         searchdata = data[pos:]
-        fname = searchdata[:searchdata.index("\0")]
-        # print "field name: ", fname, " field name len: ", len(fname)
+        print(data)
+        print(pos)
+        print(searchdata)
+        try:
+            fname = searchdata[:searchdata.index("\0")]
+        except ValueError:
+            fname = searchdata[:].strip()
```

Comment:
`fname = searchdata.strip()` ?

Reply:
I tried that at #9193703 but it didn't work